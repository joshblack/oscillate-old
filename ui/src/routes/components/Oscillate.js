import React from 'react';
import AuthProvider from './AuthProvider';

export default class Oscillate extends React.Component {
  static propTypes = {
    children: React.PropTypes.node,
  }

  render() {
    return (
      <div>
        <div>
          <h1>Oscillate</h1>
        </div>
        <AuthProvider>
          {this.props.children}
        </AuthProvider>
      </div>
    );
  }
}
