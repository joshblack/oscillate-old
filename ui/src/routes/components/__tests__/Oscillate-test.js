import React from 'react';
import renderer from 'react-test-renderer';
import Oscillate from '../Oscillate';

describe('Oscillate Component', () => {
  it('should render', () => {
    const tree = renderer.create(
      <Oscillate />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
