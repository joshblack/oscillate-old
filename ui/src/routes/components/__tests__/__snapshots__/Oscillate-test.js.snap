exports[`Oscillate Component should render 1`] = `
<div>
  <div>
    <h1>
      Oscillate
    </h1>
  </div>
  <div
    className="loader">
    <svg
      className="loader__svg"
      viewBox="-75 -75 150 150">
      <circle
        cx="0"
        cy="0"
        r="37.5" />
    </svg>
  </div>
</div>
`;
